src/drawer.multicanvas.js

```diff
@@ -332,11 +332,15 @@ WaveSurfer.util.extend(WaveSurfer.Drawer.MultiCanvas, {
 
     getImage: function(type, quality) {
         // combine all available canvasses together
-        var availableCanvas = '';
+        var availableCanvas = [];
         for (var i in this.canvases) {
```

Comment:
`this.canvases` is an array, please use `this.canvases.forEach(function (entry) {`– This is a pattern which we shouldn't use in this context. see https://github.com/katspaugh/wavesurfer.js/pull/961